float64 x
float64 y
float64 th

int64 BIPED=0
int64 TROT=1
int64 PACE=2
int64 CRAWL=3
int64 GALLOP=4

int64 gait_type
